webapp/src/insights/index.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export type TopBoard = {
+    boardID: string;
+    icon: string;
+    title: string;
+    activityCount: number;
+    activeUsers: string;
```

Comment:
Does it make sense for `activeUsers` to be a `string`?

Reply:
it's the `string_agg(user_id, ',')` value frontend uses to populate participants. It should be fixed, but in my opinion we can take some time after 7.2 release to work on this.  [created a ticket](https://mattermost.atlassian.net/browse/MM-46251)